server/views/pages/createGoals/index.njk

```diff
@@ -113,7 +113,7 @@
 
             {{ govukRadios({
               name: "goals[" + goalLoop.index0 + "][targetCompletionDate]",
-              id: "goals[" + goalLoop.index0 + "].targetCompletionDate",
+              idPrefix: "goals[" + goalLoop.index0 + "].targetCompletionDate",
```

Comment:
This fixes a small problem where the error reported at the top of the screen did not link to/jump to the relevant field.
It was only a problem for target completion date, because it's radio buttons, and we'd used the wrong attribute on the `govkRadios` macro
Fixed and working now 👍